task/01-strings-tasks.js

```diff
@@ -69,7 +69,7 @@ function getStringFromTemplate(firstName, lastName) {
  *   'Hello, Chuck Norris!' => 'Chuck Norris'
  */
 function extractNameFromTemplate(value) {
-    throw new Error('Not implemented');
+    return value.split('').splice(7, value.length - 8).join('');
 }
```

Comment:
`splice` can work with negative values(it will take symbols from end of string)
`splice(7, - 8)` do the same.
Also, the solution can be more interesting with RegExp and replace

Reply:
fix